Xcode 13 / Swift 5.5 configuration failing

Greetings.

I'd be using this library through Komondor without any issue until the last Swift 5.5 release, where PackageConfig no longer is able to compile nor generate the JSON config files needed for the other tools to work. This is the whole build:

```
CMD: /usr/bin/xcrun --find swiftc
Using SPM version: /Applications/Xcode-13.0.0-Release.Candidate.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/pm/libPackagePlugin.dylib
CMD: /Applications/Xcode-13.0.0-Release.Candidate.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swiftc --driver-mode=swift -L /Applications/Xcode-13.0.0-Release.Candidate.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/pm/libPackagePlugin.dylib -I /Applications/Xcode-13.0.0-Release.Candidate.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/pm/libPackagePlugin.dylib -lPackageDescription -package-description-version 5.5 -L .build/debug -I .build/debug -lPackageConfig -suppress-warnings Package.swift
Package.swift:4:8: error: no such module 'PackageDescription'
import PackageDescription
       ^
Finished launching swiftc
Error(reason: "Could not find a file at /var/folders/x6/dpg51cd946x8w__7znsbz5x00000gq/T/example.config.json - something went wrong with compilation step probably")
```

--- a/Sources/PackageConfig/Package.swift
+++ b/Sources/PackageConfig/Package.swift
@@ -114,7 +114,9 @@ enum Package {
         let fileManager = FileManager.default
 
         let swiftPMDir = swiftPath.replacingOccurrences(of: "bin/swiftc", with: "lib/swift/pm")
-        let versions = try! fileManager.contentsOfDirectory(atPath: swiftPMDir).filter { $0 != "llbuild" }
+        let versions = try! fileManager.contentsOfDirectory(atPath: swiftPMDir)
+            .filter { $0 != "llbuild" }
+            .filter { $0.first?.isNumber ?? false }
 
         let latestVersion = versions.sorted().last!
         var spmVersionDir = latestVersion

